mealie/routes/recipe/recipe_crud_routes.py

```diff
@@ -131,7 +132,9 @@
             if image_asset.is_file():
                 myzip.write(image_asset, arcname=image_asset.name)
 
-        return FileResponse(temp_path, filename=f"{slug}.zip")
+        return FileResponse(
+            temp_path, filename=f"{slug}.zip", background=BackgroundTask(temp_path.unlink, missing_ok=True)
```

Comment:
* It depends on the `slug` to the same degree the previous code did.  
* Slug here is not the passed in string, but the value in the encrypted jwt token.  However that encrypted value is created from user input so that's no reason to trust it.
* We are not using the slug to read or write from the mealie volume, but only to supply an advisory filename to the client.  
* We are using slug to lookup a value in the db, so the slug must match a slug in the db or we will not reach this code.
    * I have not verified, but I'm assuming sql injection from slug lookups has been mitigated. 
* Slugify should prevent the creation of any recipe title -> slug that would allow directory traversal or similar dangerous derived filenames from being sent to the user.

The change does not seem to have changed the risk profile of this function, and it appears safe to me, though if someone sees a risk here please let me know. 

There is an odd bit of code that is `if slug != slug:` which I suspect is supposed to verify that the slug in the token matches the slug passed in the path, but at present that if clause can never trigger and we just use the the slug from the token.  I can fix that comparison if people agree with my assessment of what it is supposed to be doing.